vault_program/src/slash.rs

```diff
@@ -54,31 +55,35 @@ pub fn process_slash(
     } = SanitizedAccounts::sanitize(program_id, accounts, slot)?;
 
     // The vault shall be opted-in to the AVS and the AVS shall be opted-in to the vault
-    vault_avs_ticket.vault_avs_ticket().check_active(slot)?;
-    avs_vault_ticket.avs_vault_ticket().check_active(slot)?;
+    vault_avs_ticket
+        .vault_avs_ticket()
+        .check_active(slot, config.config().epoch_length())?;
+    avs_vault_ticket
+        .avs_vault_ticket()
+        .check_active(slot, config.config().epoch_length())?;
 
     // The operator shall be opted-in to vault and the vault shall be staked to the operator
     operator_vault_ticket
         .operator_vault_ticket()
-        .check_active(slot)?;
+        .check_active(slot, config.config().epoch_length())?;
     vault_operator_ticket
         .vault_operator_ticket()
-        .check_active(slot)?;
+        .check_active(slot, config.config().epoch_length())?;
 
     // The operator shall be opted-in to the AVS and the AVS shall be opted-in to the operator
     avs_operator_ticket
         .avs_operator_ticket()
-        .check_active(slot)?;
+        .check_active(slot, config.config().epoch_length())?;
```

Comment:
check_staked for these, which would be active or cooling down

Reply:
Implemented what we talked about